doc/Makefile

```diff
@@ -209,11 +209,11 @@ doc/index.rst: $(MANPAGES:=.md)
 
 # So GitHub renamed master to main.  This is painful.
 schema-added-check:
-	@if ! git describe master >/dev/null 2>&1; then MASTER=main; else MASTER=master; fi; if git diff $$MASTER doc/schemas | grep -q '^+.*{' && ! git diff master doc/schemas | grep -q '^+.*"added"'; then echo 'New schema fields must have "added": "vNEXTVERSION"' >&2; exit 1; fi
+	@git fetch origin master ; if git diff origin/master...HEAD -- doc/schemas | grep -q '^+.*{' && ! git diff origin/master...HEAD -- doc/schemas | grep -q '^+.*"added"'; then git diff origin/master...HEAD -- doc/schemas ; echo 'New schema fields must have "added": "vNEXTVERSION"' >&2; exit 1; fi
```

Comment:
`git fetch` in a script will cause people headaches if they have a passphrase or a hardware key holding the SSH keys.

Reply:
Yep, can't do it that way... :/